.github/workflows/nvd-scan.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Set up Clojure env
-        uses: yetanalytics/actions/setup-env@14649ebc3eb8c7092a01ab7d3a576efc5c3eb9aa
+        uses: yetanalytics/actions/setup-env@v0.0.1
```

Comment:
Unfortunately it seems that nvd-scan will always be slightly behind since it can't reference its own tag. (Or maybe it can and we just accept an instance of failing CI?)

Reply:
if it breaks CI, leave it at a SHA. more important to have meaningful CI

Reply:
It doesn't break CI now; the issue is that the v0.0.1 version of nvd-scan will reference the pre-v0.0.1 version fo setup-env, while it'll be an untagged future version of nvd-scan that references v0.0.1 of setup-env.

Reply:
Wait, I'm not sure I see what the issue really is here... this code won't go into a new version until we cut one, at which point it will either still be able to point at the old or just get bumped to a new SHA.

Reply:
By "untagged future version" I meant "it doesn't have a version number, it's just a random update"

Reply:
We're not gonna express a single break ver accurately across n different tools, so if internally they point to an older sha or version that's not a problem

Reply:
Ok